packages/components/src/FormField/FormField.css

```diff
@@ -108,6 +104,11 @@
   overflow: auto;
 }
 
+.wrapper.textarea.safari {
+  overflow: unset;
+  overflow-y: scroll;
```

Comment:
using the Y axis because it feels less intrusive or disruptive

if I use the X, then it reserves the space for the horizontal scroll, and make the input feel much more imbalanced

<img width="1053" alt="image" src="https://github.com/GetJobber/atlantis/assets/11843215/9248b108-2e3d-4080-9c97-93961daf0dff">

while it's still a similar problem on the Y axis, it's much harder to notice. it's only when things go all the way to the edge that you might see that it stops earlier due to the extra spacing, or the situation where you have the inner scrollbar like mentioned somewhere on this PR

<img width="1053" alt="image" src="https://github.com/GetJobber/atlantis/assets/11843215/032820e5-040a-4ec3-b3fc-f005921851f8">